### PR TITLE
Fixed spelling and grammar for clarity

### PR DESCRIPTION
**Pull request by drewstone**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -10,25 +10,25 @@ Next-generation framework for blockchain innovation.
 
 == Description
 
-At its heart, Substrate is a combination of three technologies: WebAssembly, Libp2p and AfG Consensus. It is both a library for building new blockchains and a "skeleton key" of a blockchain client, able to synchronise to any Substrate-based chain.
+At its heart, Substrate is a combination of three technologies: WebAssembly, Libp2p, and AfG Consensus. It is both a library for building new blockchains and a "skeleton key" of a blockchain client, able to synchronise to any Substrate-based chain.
```
Comment (gavofyork) [inline README.adoc]: No Oxford comma in this house - we're readers of The Economist!

File: README.adoc [hunk 2]
```diff
@@ -12,23 +12,23 @@ Next-generation framework for blockchain innovation.
...
+Substrate chains have three distinct features that make them "next-generation": a dynamic, self-defining state-transition function, light-client functionality from day one, and a progressive consensus algorithm with fast block production and adaptive, definite finality. The STF, encoded in WebAssembly, is known as the "runtime". This defines the `execute_block` function, and can specify everything from the staking algorithm, transaction semantics, logging mechanisms and procedures for replacing any aspect of itself or of the blockchain's state ("governance"). Because the runtime is entirely dynamic all of these can be switched out or upgraded at any time. A Substrate chain is very much a "living organism".
```
Comment (gavofyork) [inline README.adoc]: , and

File: README.adoc [hunk 3]
```diff
@@ -12,23 +12,23 @@ Next-generation framework for blockchain innovation.
...
 
 == Usage
 
-Substrate is still an early stage project, and while it has already been used as the basis of major projects like Polkadot, using it is still a significant undertaking. In particular, you should have a good knowledge of blockchain concepts and basic cryptography. Terminology like header, block, client, hash, transaction and signature should be familiar. At present you will need a working knowledge of Rust to be able to do anything interesting (though eventually, we aim for this not to be the case).
+Substrate is still an early stage project, and while it has already been used as the basis of major projects like Polkadot, using it is still a significant undertaking. In particular, you should have a good knowledge of blockchain concepts and basic cryptography. Terminology like header, block, client, hash, transaction, and signature should be familiar. At present you will need a working knowledge of Rust to be able to do anything interesting (though eventually, we aim for this not to be the case).
```
Comment (gavofyork) [inline README.adoc]: , and signature

File: README.adoc (hunk 2 above)
Comment (gavofyork) [reply README.adoc]: though for this list separation by `;` would be acceptable and for that you can put it before the final "and".

File: README.adoc [hunk 4]
```diff
@@ -12,23 +12,23 @@ Next-generation framework for blockchain innovation.
...
+2. Modular: By hacking together modules from the Substrate Runtime Module Library into a new runtime and possibly altering or reconfiguring the Substrate client's block authoring logic. This affords you a very large amount of freedom over your own blockchain's logic, letting you change datatypes, add or remove modules, and, crucially, add your own modules. Much can be changed without touching the block-authoring logic (since it is generic). If this is the case, then the existing Substrate binary can be used for block authoring and syncing. If the block authoring logic needs to be tweaked, then a new altered block-authoring binary must be built as a separate project and used by validators. This is how the Polkadot relay chain is built and should suffice for almost all circumstances in the near to mid-term.
```
Comment (gavofyork) [inline README.adoc]: , and, crucially

File: README.adoc [hunk 5]
```diff
@@ -87,9 +87,9 @@ The Substrate Runtime Module Library includes functionality for timestamps and s
...
+In Substrate, there is a major distinction between blockchain *syncing* and block *authoring* ("authoring" is a more general term for what is called "mining" in Bitcoin). The first case might be referred to as a "full node" (or "light node" - Substrate supports both): authoring necessarily requires a synced node, and, therefore, all authoring clients must necessarily be able to synchronise. However, the reverse is not true. The primary functionality that authoring nodes have, which is not in "sync nodes", is threefold: transaction queue logic, inherent transaction knowledge, and BFT consensus logic. BFT consensus logic is provided as a core element of Substrate and can be ignored since it is only exposed in the SDK under the `authorities()` API entry.
```
Comment (gavofyork) [inline README.adoc]: , and, therefore

, and BFT

**Comment by drewstone** [comment]
@gavofyork How you like these commas

**Comment by gavofyork** [comment]
Merged. Thanks for the PR!

**Pull request merged**